server/src/api/account.py

```diff
@@ -24,13 +24,25 @@ def __init__(self, database):
 
     @cherrypy.expose
     def register(self):
-        """Called when client requests to register, should create the user on the user database, 
+        """Called when client requests to register, should create the user on the user table, 
             after creating a new user the client should be logged in
         
         Arguments:
             json -- check api doc for json format
```

Comment:
nit: maybe add the actual json format to the docstring now that you have implemented the endpoint?